src/main/java/com/binance/dex/api/client/Wallet.java

```diff
@@ -31,10 +33,17 @@
 
     private String chainId;
 
+    public Wallet(String privateKey, BinanceDexEnvironment env, Boolean usePreviousSequence){
+        this(privateKey,env);
+        this.usePreviousSequence = usePreviousSequence;
+    }
```

Comment:
need add the NPE check for param  of `usePreviousSequence `. Set it to `false` by default